### PR TITLE
DBZ-6603 Support for custom tags in the connector metrics

### PR DESCRIPTION
**Pull request by harveyyue**
https://issues.redhat.com/browse/DBZ-6603

### CODE REVIEW

**Comment by harveyyue** [comment]
Related PR:
cassandra: https://github.com/debezium/debezium-connector-cassandra/pull/112
db2: https://github.com/debezium/debezium-connector-db2/pull/113

**Comment by rk3rn3r** [comment]
LGTM, but let's wait for another pair of eyes like @Naros or @jpechane when they are back from holidays.

**Comment by harveyyue** [comment]
> @harveyyue LGTM. Can you please add an integration test to verify that tags are correctly registered. Maybe adding a test on `PostgresMetricsIT` is enough. Thanks!

OK

**Review by akanimesh7 — COMMENTED** [review]
Minor question on this one

File: debezium-core/src/main/java/io/debezium/metrics/Metrics.java [hunk 1]
```diff
@@ -49,7 +50,7 @@ protected Metrics(CommonConnectorConfig connectorConfig, String contextName, boo
                     "context", contextName));
```
[the remark this answers is not in the document]
Comment (harveyyue) [reply debezium-core/src/main/java/io/debezium/metrics/Metrics.java]: Done.

**Comment by akanimesh7** [comment]
Is there anything we are waiting on for this change to get merged ?

**Comment by jpechane** [comment]
@akanimesh7 Hi, I am sorry, I just got back for holidays. There is not fixed test failure in https://github.com/debezium/debezium-connector-spanner/pull/44

**Comment by akanimesh7** [comment]
I guess @harveyyue you'll need to make similar changes in tests of other repos also ?

**Comment by jpechane** [comment]
@akanimesh7 It is the only issue, other repos are passing.

**Comment by harveyyue** [comment]
@akanimesh7 @jpechane I will look spanner connector issue

**Comment by harveyyue** [comment]
> @akanimesh7 @jpechane I will look spanner connector issue

Done.

**Comment by akanimesh7** [comment]
@harveyyue Let's wait for @jpechane to approve and merge.

**Comment by jpechane** [comment]
@harveyyue Applied, thanks a lot!

**Pull request merged**